adhocracy4/static/Alert.jsx

```diff
@@ -1,11 +1,17 @@
-import React from 'react'
+import React, { useRef, useEffect } from 'react'
 import django from 'django'
 
 const Alert = ({ type, message, onClick, timer }) => {
+  const timer0 = useRef()
```

Comment:
maybe a different naming instead of timer0? 

for instance the argument `timeInMs` and the actual ref `timer`?

Reply:
yeah, good idea! `timer` and `timer0` is not a good choice

Reply:
i know we would need to change it where we use it. or you "rename" it here for instance ({ type, message, onClick, timer: timeInMs }). I admit its sketchy.

Reply:
we only use it in kosmo, right? I can change it there

Reply:
opened https://github.com/liqd/a4-kosmo/pull/203 for it

Reply:
perfect thanks!